backend/src/main/java/dev/k2/deleteXsdFile/DeleteXsdFileController.java

```diff
@@ -0,0 +1,22 @@
+package dev.k2.deleteXsdFile;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.FileNotFoundException;
+
+@RestController
+@RequestMapping("api/v1/delete")
+@RequiredArgsConstructor
+@CrossOrigin("*")
+public class DeleteXsdFileController {
+
+    private final DeleteXsdFileService service;
+
+    @DeleteMapping("{fileId}")
+    public ResponseEntity<FileDeleteResponse> deleteXsdFile(@PathVariable("fileId") Long fileId) {
```

Comment:
Should return void, because MinioFileRepository returns void.

Reply:
Response is still useful for updating the list on frontend

Reply:
Void i tak zwraca responsa.